Lib/test/test_cmd_line_script.py

```diff
@@ -613,6 +613,20 @@ def test_syntaxerror_indented_caret_position(self):
             self.assertNotIn("\f", text)
             self.assertIn("\n    1 + 1 = 2\n    ^", text)
 
+    def test_syntaxerror_multi_line_fstring(self):
```

Comment:
Perhaps
```suggestion
    def test_syntaxerror_multi_line_fstring(self):  # bpo-37433
```

Reply:
in previous patches I was told to remove the `bpo` comment as it's essentially noise (and `git blame` can retrieve the same information) -- let me know if you'd like to follow that suggestion or this one 👍

Reply:
Leave it alone for now.